content/blog/announcing-7-0-of-the-pulumi-aws-provider/index.md

```diff
@@ -0,0 +1,343 @@
+            .region("us-east-1")
+            .build());
+
+        var mainVpc = new Vpc("mainVpc", VpcArgs.builder()
+            .cidrBlock("10.0.0.0/16")
+            .build(), CustomResourceOptions.builder()
+                .provider(provider)
+                .build());
+
+        var peerVpc = new Vpc("peerVpc", VpcArgs.builder()
+            .cidrBlock("10.1.0.0/16")
+            .build(), CustomResourceOptions.builder()
+                .provider(provider)
+                .build());
+
+        var peerConnection = new VpcPeeringConnection("peerConnection", VpcPeeringConnectionArgs.builder()
+            .vpcId(mainVpc.id())
+            .peerVpcId(peerVpc.id())
+            .peerRegion("us-west-2")
+            .autoAccept(false)
+            .build(), CustomResourceOptions.builder()
+                .provider(provider)
+                .build());
+
+        var peerAcceptor = new VpcPeeringConnectionAccepter("peerAcceptor", VpcPeeringConnectionAccepterArgs.builder()
+            .region("us-west-2")
+            .vpcPeeringConnectionId(peerConnection.id())
+            .autoAccept(true)
+            .build(), CustomResourceOptions.builder()
+                .provider(provider)
+                .build());
+
+    }
+}
+
+```
+
+{{% /choosable %}}
+
+{{% choosable language yaml %}}
+
+```yaml
+name: cross-region-peering
+runtime: yaml
+
+resources:
+  provider:
+    type: pulumi:providers:aws
+    properties:
+      region: us-east-1
+  mainVpc:
+    type: aws:ec2:Vpc
+    properties:
+      cidrBlock: 10.0.0.0/16
+    options:
+      provider: ${provider}
+  peerVpc:
+    type: aws:ec2:Vpc
+    properties:
+      cidrBlock: 10.1.0.0/16
+    options:
+      provider: ${provider}
+  peerConnection:
+    type: aws:ec2:VpcPeeringConnection
+    properties:
+      vpcId: ${mainVpc.id}
+      peerVpcId: ${peerVpc.id}
+      peerRegion: us-west-2
+      autoAccept: false
+    options:
+      provider: ${provider}
+  peerAcceptor:
+    type: aws:ec2:VpcPeeringConnectionAccepter
+    properties:
+      region: us-west-2
+      vpcPeeringConnectionId: ${peerConnection.id}
+      autoAccept: true
+    options:
+      provider: ${provider}
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+See the [Enhanced Region Support](https://www.pulumi.com/registry/packages/aws/how-to-guides/aws-enhanced-region-support/) guide for more details.
+
+### Upstream Breaking Changes
+
+Pulumi AWS 7.0 ships all improvements and bug fixes of upstream versions from 6.0.0 to 6.3.0. It also contains a number of upstream breaking changes. Please refer to the [changelog](https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md) to navigate the entire list.
+
+### MaxItemsOne and Deprecations
+
+We shipped a few, small breaking changes to properties that may allow one or many items. We aligned those with the capabilities of the upstream provider and the AWS services ensure Pulumi users can access the full capabilities of the AWS platform.
```

Comment:
I'm not sure if this is worth a call out. It's something we do with each major version and doesn't really have material impact beyond a code change for the affected properties.